pkg/storegateway/indexheader/header.go

```diff
@@ -46,12 +46,10 @@ type Reader interface {
 
 type Config struct {
 	MapPopulateEnabled             bool `yaml:"map_populate_enabled" category:"experimental"`
-	StreamReaderEnabled            bool `yaml:"stream_reader_enabled" category:"experimental"`
 	StreamReaderMaxIdleFileHandles uint `yaml:"stream_reader_max_idle_file_handles" category:"experimental"`
```

Comment:
This shouldn't be experimental anymore (but advanced).

Reply:
Done in e18a97f.